features/networking/pod.feature

```diff
@@ -197,7 +197,8 @@ Feature: Pod related networking scenarios
   # @case_id OCP-23890
   @admin
   Scenario: A pod with or without hostnetwork cannot access the MCS port 22623 or 22624 on the master
-    Given evaluation of `env.master_hosts.first.local_ip` is stored in the :master_ip clipboard
+    Given I store the masters in the :masters clipboard
```

Comment:
Seems like ```env.master_hosts.first.local_ip``` no more works as env HOSTS now just take lb ip thats why i believe. Anyways this works great now.

Reply:
thanks the update. @anuragthehatter . they are LGTM. 
About the new added step for 'Given I store the masters in the :masters clipboard'. could you help also leave the run logs here?

Reply:
@zhaozhanqi ```Given I store the masters in the :masters clipboard``` is not i created. Its already there in node.rb. I created ```Internal IP of node``` step. Logs http://pastebin.test.redhat.com/824273
I ran it on OVN cluster and was supposed to fail due to Bug 1759338 so ignore the failures.

Reply:
@zhaozhanqi ?

Reply:
thanks. LGTM